### PR TITLE
Fix package problem: update prg_resources to pkgutil

### PR DESCRIPTION
**Pull request by arrownj**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,6 +17,10 @@ Handles multi-API versions of Azure Storage Data Plane originally from https://g
 
 Change Log
 ----------
+0.3.5
```
Comment (Juliehzl) [inline README.rst:20]: could we still use 0.3.4 because previous version is invalid?

Comment (arrownj) [reply README.rst:20]: I'm doubting if pypi allow to update the whl only? We can try that.

Comment (arrownj) [reply README.rst:20]: Confirmed with Jianhui, pypi does not support update existing version. So let's just use `0.3.5`.

**Comment by bebound** [comment]
This PR forgot to remove `pkg_resources` from these files: https://github.com/Azure/azure-cli/pull/27196#discussion_r1675047314

**Pull request merged**